src/Symfony/Component/Config/Tests/ConfigCacheFactoryTest.php

```diff
@@ -13,17 +13,18 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Config\ConfigCacheFactory;
+use Symfony\Component\Config\ConfigCacheInterface;
 
 class ConfigCacheFactoryTest extends TestCase
 {
-    /**
-     * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage Invalid type for callback argument. Expected callable, but got "object".
-     */
-    public function testCacheWithInvalidCallback()
```

Comment:
This test didn't make sense anymore, thanks to the `callable` type-hint of the `callback` argument.